frontend/lib/src/components/shared/StreamlitMarkdown/styled-components.ts

```diff
@@ -209,6 +212,14 @@ export const StyledStreamlitMarkdown =
           borderLeft: `${theme.sizes.borderWidth} solid ${theme.colors.lightGray}`,
         },
 
+        "b, strong": {
+          fontWeight: theme.fontWeights.bold,
+        },
+
+        small: {
```

Comment:
It looks like the only usage of `<small></small>` in our code was in ArrowTable.tsx and removed here in the PR. So, do we still want to override this or keep the browser default?

Reply:
My reasoning for moving it here was that we still allow combining HTML with markdown (via `allow_unsafe_html`) and `small` might be one of the more useful styling tags to include. Also, I wasn't 100% sure if there isn't any markdown rendering aspect that makes use of `small`

Reply:
Yeah I agree, but in this case we could also just use the browser default since we do not exactly know how people will use it anyways and we don't have a concrete usage, it feels very arbitrary

Reply:
Hmm, ok. Then let me remove it again and see if someone complains